### DISCUSSION TITLE
Attribute mapping does not work after update

### DISCUSSION THREAD

**Discussion by rebo0tt** [keycloak/keycloak#23860] [Q&A]
In the realm settings, I created the “Company” attribute in the user profile. In the federation settings with LDAP, I created a mapping with this attribute with the active directory. After updating from version 22.0.1 to any higher, this attribute stopped synchronizing with my active directory. Help me please

**Comment by jonkoops** [comment]
If you are experiencing a bug please report it in the issue tracker, and add the steps to reproduce the issue.
